compliance_checker/cf/util.py

```diff
@@ -8,9 +8,13 @@
 from importlib_resources import files
 from lxml import etree
 from netCDF4 import Dataset
+from pyudunits2 import UnitSystem
+from pyudunits2._exceptions import UnresolvableUnitException
```

Comment:
You can take this directly from `pyudunits2` (https://github.com/pelson/pyudunits2/blob/f75f1afd27d4b66ef77fc9db9301ae914111d288/pyudunits2/__init__.py#L14)

```
from pyudunits2 import UnitSystem, UnresolvableUnitException
```

Reply:
Thanks!